.gitignore

```diff
@@ -28,3 +30,5 @@ out/
 /docs/Gemfile.lock
 /docs/.bundle
 /docs/.rubydeps
+*.ipr
+*.iws
```

Comment:
These files are a legacy IntelliJ project format: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
I think it doesn't hurt to exclude them.